P/PROJ/build_tarballs.jl

```diff
@@ -90,8 +87,8 @@ dependencies = [
     # Host SQLite needed to build proj.db
     HostBuildDependency("SQLite_jll")
     Dependency("SQLite_jll")
-    Dependency("Libtiff_jll")
-    Dependency("LibCURL_jll"; compat="7.73")
+    Dependency("Libtiff_jll"; compat="~4.5.1")
```

Comment:
I think this is potentially overly restrictive, they didn't break the ABI from 4.1 to 4.3, they just break it in an unpredictable way.

Also, as usual this stuff has to be fixed first in the registry

Reply:
Ok, is `compat="4.5"` good? Done in cd47fa86b2f47ca0d51472bbd9b9dd2e0770384b